Eatery/Controllers/BRB/BRBAccountViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  BRBAccountViewController.swift
+//  Eatery
+//
+//  Created by William Ma on 9/1/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: -
+
+class BRBAccountViewController: UIViewController {
+    
+    private let account: BRBAccount
+    
+    private var tableView: UITableView!
+    
+    init(account: BRBAccount) {
+        self.account = account
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.register(BRBBalanceTableViewCell.self, forCellReuseIdentifier: "balance")
+        tableView.register(BRBHistoryTableViewCell.self, forCellReuseIdentifier: "history")
+        tableView.allowsSelection = false
+        tableView.separatorColor = .wash
+        tableView.dataSource = self
+        tableView.delegate = self
+        view.addSubview(tableView)
+        tableView.snp.makeConstraints { make in
+            make.top.edges.equalToSuperview()
```

Comment:
I believe doing `edges` already includes `top` so you prob just need `make.edges.equalToSuperview()`